Readme.md

```diff
@@ -4,14 +4,17 @@ Source for the content app running at: https://12factor.net/
 
 ## Development
 
+### Prerequisites
+
+You will need [Ruby](https://www.ruby-lang.org/en/documentation/installation/), [heroku](https://devcenter.heroku.com/articles/heroku-cli?utm_source=redirect&utm_campaign=cli)
+
+### Run
 ```sh
 bundle install
-heroku local:start
+heroku local:start &
 open http://localhost:5000
 ```
 
-## Deployment
```

Comment:
This header should stay; it's a standard section going forward, even though in this case it doesn't contain much information since deployment of the production application happens internally and beyond the view of the public.

Reply:
fair enough, its back :)